build.gradle

```diff
@@ -356,7 +356,7 @@ subprojects {
   // don't publish it
   // We also don't publish artifacts for kafka-streams-upgrade-system-tests since they are not necessary,
   // and were resulting in conflicts due to duplicate artifacts
-  def shouldPublish = !project.name.equals('jmh-benchmarks') && !(project.path.startsWith(':streams') && project.name.startsWith("upgrade-system-tests"))
+  def shouldPublish = !project.name.equals('jmh-benchmarks') && !project.name.equals('test-common') && !(project.path.startsWith(':streams') && project.name.startsWith("upgrade-system-tests"))
```

Comment:
nit: Maybe also update the comment above to explain why `test-common` is also excluded?